person.rb

```diff
@@ -1,16 +1,23 @@
+require './nameable'
+
 # class for person data
 class Person
```

Comment:
- [ ] Your Person class should inherit from the Nameable class

Reply:
Hi, thanks for the comment.

I have a question: Isn´t it exactly what is stated in line 1 ?

Thanks.

Reply:
it is something like this :point_down: 
```ruby
class Person < Nameable
```

Reply:
Thanks... just fixed it.